### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by da2x** [google/zopfli#183]
The last release was two years and two days ago. Master contains fixes for issues like #176 that would be nice to get into a release.

Ping @lvandeve.

**Comment by itsTyrion** [comment]
bump

**Comment by Mailaender** [comment]
Yes, please. https://github.com/google/zopfli/commit/7113f4e96bd26df27c46d590df95e517b966f10d features from 3 years ago are missing.

**Comment by WeurnomboTroc** [comment]
Work
